src/graphics/mesh.rs

```diff
@@ -221,7 +221,9 @@ impl MeshBuilder {
         self.polyline_inner(mode, points, true, color)
     }
 
-    fn polyline_inner<P>(
+    /// Create a new mesh for a given polyline.
+    /// The points given must be in clockwise order.
+    pub fn polyline_inner<P>(
```

Comment:
Why is this made `pub`?  Can it be `pub(crate)` instead?

Reply:
Good point, there's no reason for this to be public. I'll fix that.